armi/reactor/converters/geometryConverters.py

```diff
@@ -1243,6 +1243,10 @@ def convert(self, r):
         """
         Run the conversion.
 
+        .. impl:: Convert a one-third-core geometry to a full-core geometry.
+            :id: I_ARMI_THIRD_TO_FULL_CORE0
+            :implements: R_ARMI_THIRD_TO_FULL_CORE
```

Comment:
I love it!

But the requirement says "and back again", so I think we also need to put an `impl` tag on the method below this one: `restorePreviousGeometry()`.

And then, of course, tag a test or two for that method. (there are a couple, it looks like).